discuss/rpc.py

```diff
@@ -314,3 +316,22 @@ def request(self, block):
             raise ProtocolError("Transport-level error")
         return reply
 
+class RPCLocalClient(RPCClient):
+    # Args are for compatibility with the remote RPC; most aren't used
+    def __init__(self, server, port, auth, timeout):
+        # Used as the id field on meeting objects, so copy it in
+        self.server = server
+        # port 2100 is the default port -> use the binary
+        if port == 2100:
+            port = '/usr/sbin/disserve'
+        self.cmd = port
+
+        self.connect()
+        self.make_wrapper()
+
+    def connect(self):
+        pair = socket.socketpair(socket.AF_UNIX, socket.SOCK_STREAM)
+        subprocess.Popen([self.cmd], stdin=pair[1], close_fds=True)
+        pair[1].close()
+        fcntl.fcntl(pair[0].fileno(), fcntl.F_SETFD, fcntl.FD_CLOEXEC)
```

Comment:
Ummm... Most specifically, if we fork and exec, we shouldn't pass the socket pair into the child. I'm guessing I got that from some "here's how to do sockets in Python". It seems like probably a reasonable general practice in a library, to avoid impacting any exec'd children. *shrug* I'm certainly not devoted to keeping it.

(I, um, clearly managed to miss/forget/something this comment.)